joint_symbolic.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import itertools
+from scipy.signal import find_peaks
+
+plt.style.use('seaborn-v0_8-ticks')
+
+
+def get_peaks(signal):
+    peaks, _ = find_peaks(signal, distance=115)
+    difference = np.diff(peaks)
+    return peaks, signal[peaks], difference
+
+
+def make_empty_matrix():
+    my_permutations = [''.join(map(str, triplet)) for triplet in list((itertools.product([1, 0], repeat=3)))]
+    my_matrix = pd.DataFrame()
+    for triplet_1 in my_permutations:
+        for triplet_2 in my_permutations:
+            my_matrix.at[
+                "'" + triplet_1, "'" + triplet_2] = 0  # dash needed so that R does not remove the initial zeros
+    return my_matrix
```

Comment:
To wydaje się przekomplikowane, ale też bez komentarzy może mi brakować kontekstu po co i czemu tak